Orange/data/table.py

```diff
@@ -1873,3 +1905,31 @@ def assure_domain_conversion_sparsity(target, source):
     target.Y = match_density[conversion.sparse_Y](target.Y)
     target.metas = match_density[conversion.sparse_metas](target.metas)
     return target
+
+
+def vstack(arrs):
```

Comment:
`vstack` and `merge1d` were just moved, but are not used anywhere. Please move them back. If we were to expose them, we would need to properly document and test them.

Reply:
@janezd implemented them as part of Table.concatenate 4683cef2f314781be19d0989ddd88c5dc250936b, they have coverage.

I think I moved them here because they're only used here. I've prefixed them with `_`, similar to five other functions in the module.